### ISSUE TITLE
Sabnzbd-testing user-trouble

### ISSUE THREAD

**Issue by Eldinnie** [SynoCommunity/spksrc#2309]
Issue:
I'm trying to run SABnzbd-testing next to Sabnzbd.
The latter is working fine, but when installing -testing it can not run the package service. When installing it says cannot run package. SSH-ing in the NAS and trying to start the service neither works (see first log).
It seems there's something wrong with the user grouping for sabnzbd-testing it does not get added to the group sc-download as it should. See second log.

Model: DS212+
Architecture: 88f6282 (88f628x)
DSM Version: DSM 6.0-7321 Update 6

Logs:
`sudo /var/packages/sabnzbd-testing/scripts/start-stop-status start` 

```
Starting SABnzbd Testing ...
Password: <entered root password>
su: Permission denied
```

`sudo synogroup --get sc-download`

```
Group Name: [sc-download]
Group Type: [AUTH_LOCAL]
Group ID:   [65536]
Group Members:
0:[sabnzbd]
```

**Comment by Dr-Bean** [comment]
Follow the guidelines when reporting an issue: https://github.com/SynoCommunity/spksrc/blob/master/CONTRIBUTING.md#issue-content

**Comment by Eldinnie** [comment]
sorry, I modified the first post to follow guidelines,

**Comment by Dr-Bean** [comment]
See https://github.com/SynoCommunity/spksrc/issues/2191, #2207 and #2216.

**Comment by Eldinnie** [comment]
Thank you, following this with interest.

**Comment by ymartin59** [comment]
I wonder if there is still any interest in keeping "*-testing" packages (sabnzbd-testing and nzbget-testing), seeing there versions/revisions are older than "stable" ones (at least on master). Are branches not enough for this purpose ? Why not simply publish packages in a "testing" branch as "beta" any user may install if willing to ?

**Comment by Dr-Bean** [comment]
@ymartin59 The -testing packages focus purely on testing releases from upstream, per popular demand. While those versions might _currently_ not be the newest, that's not the norm.

**Comment by Safihre** [comment]
Closing due to age. Test-packages for DSM6.1 and new architectures available here: #3138.
Those will later be published to the real package rep.
